prototype/p2p/messenger_test.go

```diff
@@ -0,0 +1,78 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"testing"
+)
+
+func TestMessager_Protobuff(t *testing.T) {
+	nonce, src, dst, level := int32(1), "10.0.0.1:1234", "9.0.0.2:5432", int32(2)
+
+	// test message instantiation
```

Comment:
I think we should have different unit tests rather than sub-scoping everything within a single test.

Reply:
What do you mean?